File: model/src/main/java/no/nav/pensjon/vtp/testmodell/organisasjon/Navn.java
package no.nav.pensjon.vtp.testmodell.organisasjon;

import java.util.List;

import com.fasterxml.jackson.annotation.JsonProperty;

public class Navn {
    @JsonProperty("navnelinje")
    private List<String> navnelinje;

    public List<String> getNavnelinje() {
        return navnelinje;
    }

    public void setNavnelinje(List<String> navnelinje) {
        this.navnelinje = navnelinje;
    }
}

File: vtp-pensjon-application/src/main/kotlin/no/nav/pensjon/vtp/configuration/graphql/model/Navn.kt
package no.nav.pensjon.vtp.configuration.graphql.model

import java.util.*

data class Navn(
    val fornavn: String,
    val mellomnavn: String?,
    val etternavn: String,
    val forkortetNavn: String?,
    val gyldigFraOgMed: Date?,
    val metadata: Metadata
)